Datadog/TargetSupport/DatadogIntegrationTests/DatadogIntegrationTests.xctestplan

```diff
@@ -137,6 +137,22 @@
       {
         "key" : "BITRISE_BUILD_URL",
         "value" : "$(BITRISE_BUILD_URL)"
+      },
+      {
+        "key" : "DD_APPLICATION_KEY",
+        "value" : "$(DD_SDK_SWIFT_TESTING_APPLICATION_KEY)"
+      },
+      {
+        "key" : "DD_CIVISIBILITY_GIT_UPLOAD_ENABLED",
+        "value" : "1"
+      },
+      {
+        "key" : "DD_CIVISIBILITY_ITR_ENABLED",
+        "value" : "1"
+      },
```

Comment:
How will ITR work for integration tests? Does it learn what is typical code coverage linked with particular test and only run it if there was a code change in this scope? Would it be exactly the same ITR model as for unit tests, right?

Reply:
Yes, it works the same for all kind of tests: it captures the code coverage for the test when it runs and only runs again if any of the files that is included in the coverage changes (or if the branch is any of the excluded branches, that it runs all tests)

Reply:
One of the useful things of using the intelligent test runner is that it would allow to run more complete and potentially expensive tests more frequently because it would skip those not related